src/components/Button/Button.scss

```diff
@@ -0,0 +1,19 @@
+@use "../../styles/variables" as *;
+
+.Button {
+  &--default {
+    width: 19.4375rem;
+    height: 4.125rem;
```

Comment:
I would make these the default styles rather than use a modifier to apply them.

"Given that I want to use the button with default styles, When I include the button component without specifying any styles, Then the button should have a default style that is consistent with the application's design guidelines."

This means that the `variant` prop can be optional and the default does not need to be passed every time.